<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Translator</class>
 <widget class="QMainWindow" name="Translator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translator</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="langComboBox">
      <item>
       <property name="text">
        <string>Malayalam to English</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>English to Malayalam</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="source_text">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="target_text"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Convert_file"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Preferences"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_PLay"/>
    <addaction name="action_Translate"/>
   </widget>
   <widget class="QMenu" name="menu_Developer">
    <property name="title">
     <string>&amp;Developer</string>
    </property>
    <widget class="QMenu" name="menu_Add_New_Rule">
     <property name="title">
      <string>&amp;Add New Rule</string>
     </property>
     <addaction name="actionMal_eng"/>
     <addaction name="actionEng_mal"/>
    </widget>
    <addaction name="menu_Add_New_Rule"/>
    <addaction name="action_Compile"/>
    <addaction name="action_Add_New_Word"/>
    <addaction name="action_Add_a_BiDix_entry"/>
    <addaction name="action_Add_paradigm"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Developer"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>12</height>
    </size>
   </property>
   <property name="statusTip">
    <string extracomment="main toolbar"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Exit"/>
   <addaction name="action_Translate"/>
   <addaction name="action_PLay"/>
   <addaction name="action_About"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/icons/document-new.png</normaloff>:/new/prefix1/icons/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/icons/document-import.png</normaloff>:/new/prefix1/icons/document-import.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Convert_file">
   <property name="text">
    <string>&amp;Convert file</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/icons/application-exit.svg</normaloff>:/new/prefix1/icons/application-exit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/icons/help-about.svg</normaloff>:/new/prefix1/icons/help-about.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_PLay">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/icons/media-playback-start.svg</normaloff>:/new/prefix1/icons/media-playback-start.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;PLay</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Translate">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/icons/poedit.png</normaloff>:/new/prefix1/icons/poedit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Translate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionMal_eng">
   <property name="text">
    <string>mal-eng</string>
   </property>
  </action>
  <action name="actionEng_mal">
   <property name="text">
    <string>eng-mal</string>
   </property>
  </action>
  <action name="action_Compile">
   <property name="text">
    <string>&amp;Compile</string>
   </property>
  </action>
  <action name="action_Add_New_Word">
   <property name="text">
    <string>&amp;Add New Word</string>
   </property>
  </action>
  <action name="action_Add_a_BiDix_entry">
   <property name="text">
    <string>&amp;Add a BiDix entry</string>
   </property>
  </action>
  <action name="action_Add_paradigm">
   <property name="text">
    <string>&amp;Add paradigm</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
